### DISCUSSION TITLE
Suitable Patterns

### DISCUSSION THREAD

**Discussion by AyeshaGavandi** [Green-Software-Foundation/patterns#205] [Q&A]
Which pattern is suitable for salesforce package ?

**Comment by holanita** [comment]
@AyeshaGavandi thanks for your question 
@greenhsu123 please respond or delegate to the right SME if needed

**Comment by greenhsu123** [comment]
Hi @AyeshaGavandi, thank you for reaching out. Currently, we are keeping Patterns vendor-neutral as per https://patterns.greensoftware.foundation/#principles-patterns-and-practices. Once we have kicked off the Practices workstream, we will let you know. Thanks again!

**Reply by AyeshaGavandi** [reply]
Thanks greenhsu123!!
